glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt

```diff
@@ -397,15 +397,15 @@ class LabeledMetricTypeTest {
 
     @Test(expected = IllegalStateException::class)
     fun `Test that labeled events are an exception`() {
-        val eventMetric = EventMetricType<NoExtraKeys>(
+        val eventMetric = EventMetricType<NoReasonCodes>(
```

Comment:
Did you mean to change this here?  I think `NoExtraKeys` is correct for an `EventMetricType`.

Reply:
Oops.  Fixed now.